backends/graphics/opengl/texture.cpp

```diff
@@ -795,13 +795,12 @@ void TextureCLUT8GPU::updateGLTexture() {
 
 void TextureCLUT8GPU::lookUpColors() {
 	// Setup pipeline to do color look up.
-	Pipeline *oldPipeline = Pipeline::setPipeline(_clut8Pipeline);
+	_clut8Pipeline->activate();
 
 	// Do color look up.
-	Pipeline::getActivePipeline()->drawTexture(_clut8Texture, _clut8Vertices);
+	_clut8Pipeline->Pipeline::drawTexture(_clut8Texture, _clut8Vertices);
```

Comment:
Is there a reason for including `Pipeline::` in the function call?

Reply:
Yes, because `CLUT8Pipeline::drawTexture(const GLTexture &texture, const GLfloat *coordinates, const GLfloat *texcoords)` hides `Pipeline::drawTexture(const GLTexture &texture, const GLfloat *coordinates)`.
This is known as name hiding: https://bastian.rieck.me/blog/posts/2016/name_hiding_cxx/.

Reply:
Would it be possible to add `using Pipeline::drawTexture;` to the relevant subclasses?

Reply:
I would tend to say that these functions are not virtual so not supposed to be inherited.
Maybe we simply should rename the `virtual drawTexture` to something else as it's the one which should be overriden.
What do you think?

Reply:
I did it and it's cleaner like that.